bundles/binding/org.openhab.binding.zwave/database/domitech/domezbulb.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+	<Model>ZB22UK</Model>
+	<Label lang="en">Domitech Smart LED Light Bulb</Label>
+	<CommandClasses>
+		<Class><id>0x20</id></Class>
+		<Class><id>0x26</id></Class>
+		<Class><id>0x27</id></Class>
+		<Class><id>0x59</id></Class>
+		<Class><id>0x5a</id></Class>
+		<Class><id>0x5e</id></Class>
+		<Class><id>0x70</id></Class>
+		<Class><id>0x72</id></Class>
+		<Class><id>0x73</id></Class>
+		<Class><id>0x7a</id></Class>
+		<Class><id>0x85</id></Class>
+		<Class><id>0x86</id></Class>
+	</CommandClasses>
+	<Configuration>
+		<Parameter>
+			<Index>1</Index>
+			<Type>list</Type>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Item>
+				<Value>0</Value>
+				<Label lang="en">Full brightness</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Previous dim level</Label>
+			</Item>
+			<Label lang="en">Dim level when the light bulb is turned On</Label>
+			<Help lang="en">0: The brightness level will be resume to full brightness when the dt Smart Bulb is turned ON every time. 1: The brightness level will maintain same dim level as the last setting when the dt Smart Bulb is turned ON again.</Help>
```

Comment:
Can I suggest escaping this with the CDATA tag and adding a line break. Also, having 0 and 1 as the options in the help doesn't actually help since the user will see the text - not the values.